financial-services-accelerator/components/org.wso2.financial.services.accelerator.common/src/main/java/org/wso2/financial/services/accelerator/common/config/FinancialServicesConfigParser.java

```diff
@@ -787,4 +787,47 @@ public String getRealtimeEventNotificationRequestGenerator() {
                 "realtime.service.DefaultRealtimeEventNotificationRequestGenerator");
     }
 
+    /**
+     * Method to determine service extensions feature is enabled or not from the configurations.
+     *
+     * @return boolean value indicating the state
+     */
+    public boolean isServiceExtensionsEnabled() {
+
+        Optional<String> config = getConfigurationFromKeyAsString(
+                FinancialServicesConstants.SERVICE_EXTENSIONS_ENABLED);
+        return config.map(Boolean::parseBoolean).orElse(false);
+    }
+
+    /**
+     * Method to get service extensions base url.
+     *
+     * @return String service extensions base url
+     */
+    public String getServiceExtensionsBaseUrl() {
+
+        Optional<String> config = getConfigurationFromKeyAsString(
+                FinancialServicesConstants.SERVICE_EXTENSIONS_BASE_URL);
+        return config.map(String::trim).orElse(null);
+    }
+
+    /**
+     * Method to get enabled service extension types.
+     *
+     * @return List of enabled service extensions
+     */
+    public List<String> getEnabledServiceExtensionTypes() {
```

Comment:
Shall we make this a list of enum?

Reply:
fixed with 29599cdecba6e48ed7bee3449da68d105d20ed42